test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -36,17 +39,32 @@ auto mk_local_peer(uint64_t num) {
 class FixedCryptoProvider : public MockYacCryptoProvider {
  public:
   explicit FixedCryptoProvider(const std::string &public_key) {
-    pubkey.fill(0);
-    std::copy(public_key.begin(), public_key.end(), pubkey.begin());
+    auto tmp =
+        shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()
+            .publicKey();
+    std::string key(tmp.blob().size(), 0);
+    std::copy(public_key.begin(), public_key.end(), key.begin());
+    pubkey = clone(shared_model::crypto::PublicKey(key));
   }
 
   VoteMessage getVote(YacHash hash) override {
     auto vote = MockYacCryptoProvider::getVote(hash);
-    vote.signature.pubkey = pubkey;
+    shared_model::builder::DefaultSignatureBuilder()
+        .publicKey(shared_model::crypto::PublicKey(*pubkey))
+        .signedData(shared_model::crypto::Signed(""))
```

Comment:
Might be broken after adding stateless validation for that, plz fix that

Reply:
Yes but fix it with test signature builder. Data should be a stub.